Makefile

```diff
@@ -781,10 +797,20 @@ COMMON_SRC = \
             io/vtx_string.c \
             io/vtx_smartaudio.c \
             io/vtx_tramp.c \
-            io/vtx.c \
+            io/vtx.c
+            
+COMMON_DEVICE_SRC = \
             $(CMSIS_SRC) \
             $(DEVICE_STDPERIPH_SRC)
 
+ifeq ($(OSD_SLAVE),yes)
+TARGET_FLAGS := -DUSE_OSD_SLAVE $(TARGET_FLAGS)
+COMMON_SRC := $(COMMON_SRC) $(OSD_SLAVE_SRC) $(COMMON_DEVICE_SRC)
+else
+TARGET_FLAGS := -DUSE_FC $(TARGET_FLAGS)
```

Comment:
agreed that `USE_NON_FC` or `NON_FC_TARGET` is not good because of the double negative issue.  I deliberated on that myself for a while and settled on the two positive variations.

I'm happy to use `#ifndef USE_OSD_SLAVE` instead of `#ifdef USE_FC` though i prefer avoiding `#ifndef` in general, finding a balance is tricky.  We have similar issues with the `SKIP_*` defines.